codebuild/scripts/package_operators.sh

```diff
@@ -34,23 +34,18 @@ function deploy_from_alpha()
   # Login to the prod repository
   $(aws ecr get-login --no-include-email --region ${account_region} --registry-ids ${account_id})
 
-  # Clone the controller image to the repo and set as latest
+  # Clone the controller image to the repo
   docker tag $alpha_ecr_image:$CODEBUILD_RESOLVED_SOURCE_VERSION ${dest_ecr_image}:$CODEBUILD_RESOLVED_SOURCE_VERSION
-  docker tag $alpha_ecr_image:$CODEBUILD_RESOLVED_SOURCE_VERSION ${dest_ecr_image}:latest
```

Comment:
We should keep this `latest` tag until we have `v2`. Otherwise, customers with operators installed before your changes will stop getting minor version updates.

Reply:
+1